.dockerignore

```diff
@@ -1,5 +1,4 @@
 **/.gitignore
-.*
```

Comment:
Why removing this?

Reply:
* to pull in the rustflags from the [.cargo/config](https://github.com/oasislabs/oasis-chain/blob/master/.cargo/config#L2)
* to get the git rev for the [build script](https://github.com/oasislabs/oasis-chain/blob/master/build.rs)